src/test/java/bookmarked/command/ReturnCommandTest.java

```diff
@@ -1,127 +1,104 @@
 package bookmarked.command;
 
 import bookmarked.Book;
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.PrintStream;
-import java.time.LocalDate;
-import java.util.ArrayList;
-
 import bookmarked.user.User;
-import org.junit.jupiter.api.AfterEach;
+import bookmarked.exceptions.EmptyListException ;
+import bookmarked.exceptions.EmptyArgumentsException ;
+import bookmarked.exceptions.BookNotFoundException ;
+import bookmarked.exceptions.UserNotFoundException ;
+import bookmarked.exceptions.IndexOutOfListBounds;
+import java.time.Period;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import java.time.Period;
+import org.junit.jupiter.api.AfterEach;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.time.LocalDate;
+import java.util.ArrayList;
 
 public class ReturnCommandTest {
-    private static final String TEST_BOOK_FILE_PATH = "./testBooks.txt";
-    private static final String TEST_USER_FILE_PATH = "./testUsers.txt";
     private ArrayList<Book> listOfBooks;
-    private ArrayList<User> listOfUser;
-    private File bookDataFile;
-    private File userDataFile;
-    private Book borrowedBook;
-    private Book notBorrowedBook;
-    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
-    private final PrintStream originalOut = System.out;
+    private ArrayList<User> listOfUsers;
+    private File dummyBookDataFile;
+    private File dummyUserDataFile;
 
     @BeforeEach
     public void setUp() {
         listOfBooks = new ArrayList<>();
-        listOfUser = new ArrayList<>();
-
-        // Assume bookDataFile is either a mock or set to a temporary/testing file path
-        bookDataFile = new File(TEST_BOOK_FILE_PATH);
-        userDataFile = new File(TEST_USER_FILE_PATH);
-
-        borrowedBook = new Book("Borrowed Book");
-        // Simulate that the book has been borrowed
-        borrowedBook.borrowBook(LocalDate.now(), Period.ofDays(14));
-        notBorrowedBook = new Book("Not Borrowed Book");
-
-        listOfBooks.add(borrowedBook);
-        listOfBooks.add(notBorrowedBook);
-
-        System.setOut(new PrintStream(outContent)); // Redirect System.out to capture console output
+        listOfUsers = new ArrayList<>();
+        dummyBookDataFile = new File("dummyBookData.txt"); // Assume these are set up for testing
```

Comment:
i think better avoid directly writing the path, but instead use constant like `TEST_BOOK_FILE_PATH`?